tests/gpu/misc/examples_tf.R

```diff
@@ -0,0 +1,36 @@
+
+## Tensorflow:
+install.packages('keras', repos='http://cran.us.r-project.org')
```

Comment:
Is there any reason to use a US CRAN mirror?
https://github.com/rocker-org/rocker-versioned2/blob/a382d7da54d907e81efd007aa2c49f36197bbb1b/scripts/install_R.sh#L9

Reply:
@RobJY is this to trigger source installation?

Reply:
Should we need this line at all since tensorflow is installed in the containers by `scripts/install_tensorflow.sh`?
When I attempt to run `exampls_tf.R` without it though I get the following error:
```
Error in library(keras) : there is no package called ‘keras’
Execution halted
```
Running `library(tensorflow)` in R gives a similar error message.

Running `scripts/test-config-cuda.sh` reports the correct tensorflow version, but it's checking the version from Python with `python -c 'import tensorflow as tf; print(tf.__version__)'`.

Do I need to add the path where tensorflow is installed by `scripts/install_tensorflow.sh` somewhere so R sees it?

Reply:
What I wanted to point out is that the `repos` argument may simply be unnecessary.

Reply:
Yes @eitsupi, you're right.  It worked fine when I removed `repos`.  Thanks!  I've committed that change.

It still seems like it shouldn't need to install that though.  I tried adding the path `/opt/venv/reticulate` with the following code, but I got the same error:
```
old_path <- Sys.getenv("PATH")
Sys.setenv(PATH = paste(old_path, "/opt/venv/reticulate", sep = ":"))
```

Is it fine to install `keras` here or does the fact that I need to run it indicate there's an issue with the tensorflow install?